### PR TITLE
[UI/Pattern] Fix pattern-upload failure

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
Signed-off-by: Abhishek-kumar09 <abhimait1909@gmail.com>

**Description**
1. Fix pattern-upload failures by url

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Chadha93 — APPROVED** [review]
Cool :100:

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -155,7 +157,11 @@ function YAMLEditor({ pattern, onClose, onSubmit }) {
           <IconButton
             aria-label="Delete"
             color="primary"
-            onClick={() => onSubmit(yaml, pattern.id, pattern.name, FILE_OPS.DELETE)}
+            onClick={() => onSubmit({
+              data : yaml,
+              id : pattern.id,
+              name : pattern.name,
```
Comment (Abhishek-kumar09) [inline ui/components/MesheryPatterns.js:163]:
```suggestion
              name : pattern.name,
              action: FILE_OPS.DELETE
```

**Review by warunicorn19 — DISMISSED** [review]
Checked locally, both upload via URL and local file are working

**Review by warunicorn19 — APPROVED** [review]
@Abhishek-kumar09 merge at-will

**Comment by leecalcote** [comment]
Reverting eslint update in #5316 to address UI integration test failure.

**Pull request merged**